src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2SecurityTokenHandler.cs

```diff
@@ -269,7 +269,9 @@ private ClaimsPrincipal ValidateToken(Saml2SecurityToken samlToken, string token
                 identity.BootstrapContext = samlToken.Assertion.CanonicalString;
 
             if (LogHelper.IsEnabled(EventLogLevel.Informational))
-                LogHelper.LogInformation(TokenLogMessages.IDX10241, token);
+                LogHelper.LogInformation(
```

Comment:
names can be simplified (ex. `LogHelper` can be removed) and do we have tests verifying this behavior? same w/other similar areas

Reply:
Library doesn't have per log line testing. I've filled the testing gap for LogHelper to some extent but adding per log tests is outside the scope of this PR.